R/runner.R

```diff
@@ -348,6 +354,7 @@ orderly_runner_ <- R6::R6Class(
         key = key,
         status = out_status,
         version = report_id,
+        start_time = start_time,
```

Comment:
Two things, possibly for later:

* we should make sure to store this start time information outside of rrq if we want to persist it
* did we want to return finish time in the case where this is not still running?

Reply:
@EmmaLRussell has mentioned persisting the start time in the front end, not sure if that is sensible?

For the finish time we return the time in the `report/info` endpoint. This is the end time stored by orderly in the rds. Can return the rrq end time too if @EmmaLRussell has any need for it? I'm guessing it will be slightly different to the orderly time which might be a bit confusing

Reply:
I don't think we've got any use for the finish time at the moment. I think for finished reports, the elapsed time is more importnant and that's already in report_version table?

I suppose the problem with storing start time in the front end is that we won't have that data unless we specifically ask for it, which we won't do if the user hasn't browsed to the workflow progress page. Could it go into report_version too?

Reply:
No easy way to get it into `report_version`, no.  Within that table though there is `date` which is finishing time I think, and you have `elapsed` which is seconds it ran from - you can compute a starting time from that! Or if you wanted to get _really_ nasty the id can be parsed easily into a time.

Reply:
OK, so could try to get it from the report status endpoint, and if it's not there, we might find the endtime in report_version, from which we can work out the start time, and that's likely to cover most cases..